components/image/image.css

```diff
@@ -0,0 +1,14 @@
+.img-responsive{
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+}
+
+.image-size-container{
+    border: 2px solid var(--primary-color);
+    padding: 2rem;
+}
+.img-round{
+    border-radius: 50%;
+    overflow: hidden;
```

Comment:
You have to also give the width and height to the image to make it round.
Currently you are getting images from picsum that is why the image is of same width and height but when you will try this code on any other image you would not get round image.

So you have to give same width and height to the image to make it round.

Reply:
Ok would fix it!